SECURITY.md

```
@@ -0,0 +1,24 @@
+Security Policy
+Supported Versions
+The following versions of PlayCafe are currently supported with security updates:
+
+Version	Supported
```

Markdownlint

Column: 8
Hard tabs

(MD010, no-hard-tabs)